SplunkRumWorkspace/SplunkRum/SplunkRum/Constants.swift

```diff
@@ -0,0 +1,78 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+struct Constants {
```

Comment:
Just a comment here. I don't necessarily think it needs a change, but I think it is better to use an enum over a struct when the purpose is to namespace some group of constants like this. It makes it so it is impossible to ever instantiate the type `Constants`. A enum can have no cases and just serve as a namespace!

Reply:
a private init would also work, of course 👍